src/beans/fetchCourseCritique.js

```diff
@@ -0,0 +1,79 @@
+import cheerio from 'cheerio';
+import $ from 'jquery';
```

Comment:
jQuery is usually discouraged to be used with a front-end framework like React or Angular. If you only used it for fetching data, I recommend you to use [axios](https://github.com/axios/axios) instead.

https://github.com/64json/gt-scheduler/blob/1d5d8a715b55b1b14141126cab151fc04087f911/src/components/App/index.jsx#L34